checkov/common/bridgecrew/platform_integration.py

```diff
@@ -422,8 +423,12 @@ def persist_enriched_secrets(self, enriched_secrets: list[EnrichedSecret]) -> st
                           f' repo_path={self.repo_path}, bucket={self.bucket}')
             return None
 
-        repo_path_without_src = os.path.dirname(self.repo_path)
-        s3_path = f'{repo_path_without_src}/{checkov_results_prefix}/{CheckType.SECRETS}/secrets_to_verify.json'
+        if not bc_integration.bc_api_key or not os.getenv("CKV_VALIDATE_SECRETS"):
+            logging.debug('Skipping persistence of enriched secrets object as secrets verification is off,'
+                          ' enabled it via env var CKV_VALIDATE_SECRETS and provide an api key')
+            return None
+
+        s3_path = f'checkov/original_secrets/{uuid.uuid4()}.json'
```

Comment:
let's talk about this

Reply:
changed the path, example: `original_secrets/elturgeman5/elturgeman5_cli_repo/verifier/1674031834070/686fd705-c869-4ed3-9b07-528ff5397013.json`